src/aac/ui/menu_bar.py

```diff
@@ -0,0 +1,65 @@
+from tkinter import Menu, Tk
+
+
+def donothing():
+    print("Do nothing!")
+
+
+def menu_bar(root_window: Tk) -> Menu:
+    """
+    Creates and returns the AaC UI menu bar.
+
+    Args:
+        root_window (Tk): the window to attach the menu to
+
+    Returns:
+        The menu bar.
+    """
+    menu_bar = Menu(root_window, tearoff=0)
+    _add_file_menu(menu_bar)
+    _add_edit_menu(menu_bar)
+    _add_view_menu(menu_bar)
```

Comment:
We spoke about removing the "View" menu but I think we could potentially use it for some/all of the following items (even if they're just disabled until they're implemented):

I know the first two are possible other places but I could see it being useful to be able to do it from the app menu. Either way we go with those, I think the last would be a good use for the view menu.

- We can use it as another place someone could switch from diagram to textual view - i.e. for power users that want to directly edit YAML.
- Change between "Diagram", "Properties", and "Imports" tabs.
- (Long-range but useful, IMHO) Change the types of diagram being displayed - i.e. app architecture, deployment architecture, network architecture, etc